Subtitle 1.srt

```diff
@@ -0,0 +1,31 @@
+1
+01:00:18,625 --> 01:00:20,041
+<b>My name is Lucy Nyirenda</b>
+
+2
+01:00:21,208 --> 01:00:23,291
+<b>I stay at Mapalwe Village, Senior group Msiwa</b>
+
+
+3
+01:00:25,291 --> 01:00:27,083
+<b>I thank K2 TASO AND B.T.M</b>
+
+4
+01:00:27,958 --> 01:00:31,333
+<b>For donating  Chlorine that help fight diarhiea disease</b>

```

Comment:
```suggestion
<b>for donating chlorine to help fight diarrheal disease</b>

```

Reply:
Dee suggested `infectious diarrheal disease`, what do you think? It might be clearer but not a direct translation of what was said.